packages/arb-token-bridge-ui/src/util/EthDepositUtils.ts

```diff
@@ -56,36 +72,47 @@ export async function depositEthEstimateGas(
   } = params
   const ethBridger = await EthBridger.fromProvider(childChainProvider)
 
-  const customFeeToken = typeof ethBridger.nativeToken !== 'undefined'
-
-  if (customFeeToken && (await customFeeTokenAllowanceIsInsufficient(params))) {
-    return fetchFallbackGasEstimatesForOrbitChainWithCustomFeeToken()
-  }
+  const customFeeToken =
+    typeof ethBridger.nativeToken !== 'undefined' &&
+    ethBridger.nativeToken !== constants.AddressZero
 
   const isDifferentDestinationAddress = isCustomDestinationAddressTx({
     sender: address,
     destination: destinationAddress
   })
 
-  if (isDifferentDestinationAddress) {
-    const depositToRequest = await ethBridger.getDepositToRequest({
-      amount,
-      from: address,
-      parentProvider: parentChainProvider,
-      childProvider: childChainProvider,
-      // we know it's defined
-      destinationAddress: String(destinationAddress)
+  if (customFeeToken && (await customFeeTokenAllowanceIsInsufficient(params))) {
```

Comment:
I think `customFeeTokenAllowanceIsInsufficient` could return true when it shouldn't, because we are checking against `amount`, not including retryable fees